pkg/websocket/client.go

```diff
@@ -437,8 +437,18 @@ func (c *Client) writePump() {
 				if ws.IsUnexpectedCloseError(err, ws.CloseNormalClosure) {
 					c.cfg.Log.Error("write error: ", err)
 				}
-				c.notifyClose <- err
 
+				// writing to notifyClose during
```

Comment:
did you forget to finish the comment?

Reply:
I'd add a quick mention about that deadlock we're fixing

Reply:
ha! Yes. Yes, I did.